doc/guides/device_mgmt/dfu.rst

```diff
@@ -48,5 +48,5 @@ More detailed information regarding the use of MCUboot with Zephyr  can be found
 in the `MCUboot with Zephyr`_ documentation page on the MCUboot website.
 
 .. _MCUboot boot loader: https://mcuboot.com/
-.. _MCUboot with Zephyr: https://mcuboot.com/documentation/readme-zephyr/
```

Comment:
I can load https://mcuboot.com/documentation/readme-zephyr/ (without www).

Reply:
Does it matter if it has www in front?

Reply:
As soon as their webserver handles that, no. It seems to work without in this case. Most websites work with or without www nowadays.